packages/cashier/src/Stores/Cashier/cashier-store.js

```diff
@@ -235,8 +235,9 @@ export default class CashierStore extends BaseStore {
                 // wait for get_settings so is_virtual gets populated in client-store
                 await this.WS.wait('get_settings');
```

Comment:
if we are removing is_virtual then we don't need this line anymore either, please check

Reply:
Hi, thank you for the suggestion, I checked the behavior without this line but it seems that the old issue with user getting logged out after changing accounts happens again. Seems like the application still has to wait for other properties to be fetched as well (including `is_logged_in`), so I updated the comment to reflect this.